### DISCUSSION TITLE
How can I give windowrules to a named Special workspace

### DISCUSSION THREAD

**Discussion by Sohail-XD** [hyprwm/Hyprland#2584] [Q&A]
Default special workspace is working fine `windowrule=workspace Hello,kitty`
But I named a special workspace with `bind = ALT_SHIFT, D, movetoworkspacesilent, special Hello`. Trying to assign this "Hello" workspace to a window is not working.

**Comment by otakutyrant** [comment]
How did you assign the special workspace name? It seems that `renameworkspace` can only assign names to ids, but not `special`.

**Comment by YarochkinAnton** [comment]
Looks like you should use `special:Hello` as workspace id in config

**Comment by Nukesor** [comment]
Although this is a rather old post, I'll just write my current config windowrule that works :)

```
windowrule = float, workspace:name:special:s1, class:(Alacritty)
windowrule = center, workspace:name:special:s1, class:(Alacritty)
```

This makes any alacritty instances on the special workspace `s` float in the center of the screen :blush:. Hope it helps.

Note that multiple actions (float, center) don't trigger when in the same line.
